lighthouse-core/config/plots-config.js

```diff
@@ -6,20 +6,13 @@
 'use strict';
 
 module.exports = {
```

Comment:
Looks like we don't need our own config anymore 💃 
https://github.com/paulirish/pwmetrics/blob/v3.2.0/lib/lh-config.ts